xtask/codegen/css.ungram

```diff
@@ -1259,6 +1316,7 @@ CssSimpleFunction =
 	items: CssParameterList
 	')'
 
+// TODO add 'src'
 CssUrlFunction = 'url' '(' AnyCssUrlValue? ')'
```

Comment:
I think long term we might want to look at making all of these functions their own node types (like `CssSrcFunction`, `CssCalcFunction`, `CssSupportsFunction`, etc.), since they seem to have specific grammars, and also there isn't any whitespace allowed between the name and the `(`. I feel like that might either mean we make the lexer not skip over whitespace by default like you were thinking a while ago, or we do the same kind of hack as the `CSS_DIMENSION_VALUE` stuff but for like `CSS_FUNCTION_NAME` or something.

Reply:
I've been thinking about a special parser state which doesn't skip whitespace and having a function which have the same signature as a `try_parse`. It can set the state and rollback it after a callback. What do you think?